### PR TITLE
Codemod unnamed default exports to have names

### PR DESCRIPTION
**Pull request by akx**
This makes debugging that much easier when functions have names instead of just being _anonymous_. This is a prelude for an, ahem, slightly larger changeset.

Also, many files had been left untouched by `pretty-quick`/Prettier before, so the first commit runs that so the diff for the codemod commit is smaller.

The `jscodeshift` script I used was facebook/create-react-app#8924 (comment)

### CODE REVIEW

**Comment by danburzo** [comment]
> Also, many files had been left untouched by pretty-quick/Prettier before, so the first commit runs that so the diff for the codemod commit is smaller.

Yeah, I might have accidentally bumped up the `prettier` version in one of the dependency upgrade batches. No worries!

File: src/random.js [hunk 1]
```diff
@@ -32,3 +32,6 @@ export default (mode = 'rgb', constraints = {}) => {
 		{ mode }
 	);
 };
+
+// FIXME: identifier "random" has already been declared or isn't helpful, rename this please!
```
Comment (danburzo) [inline src/random.js:36]: Any reason why `random` does not work here?

File: src/round.js [hunk 2]
```diff
@@ -3,5 +3,8 @@
 const r = (value, precision) =>
 	Math.round(value * (precision = Math.pow(10, precision))) / precision;
 
-export default (precision = 4) => value =>
+const round1 = (precision = 4) => value =>
 	typeof value === 'number' ? r(value, precision) : value;
+
+// FIXME: identifier "round" has already been declared or isn't helpful, rename this please!
+export default round1;
```
Comment (danburzo) [inline src/round.js:10]: This should similarly be fixed

**Comment by danburzo** [comment]
Should we also add the `eslint` plugin to enforce this rule going forward?

**Comment by akx** [comment]
@danburzo Fixed the FIXMEs, I honestly hadn't even noticed them. Might've been a bug in the codemod script – good catch anyway!

Re the Eslint plugin, yeah, sounds like a good idea... I tried looking through https://github.com/benmosher/eslint-plugin-import 's readme but couldn't find a matching rule. I can set the tooling up if you give me a pointer as to what to do :)

**Comment by danburzo** [comment]
Thanks! I'll take a stab at it with the eslint plugin, many of the rules there may be useful.

**Comment by danburzo** [comment]
I've enabled a couple of rules from the `import` plugin here: https://github.com/Evercoder/culori/commit/36c9598c20dd8c8efd142ae6e2fe0a0b5e701037

**Comment by danburzo** [comment]
> this is a prelude for an, ahem, slightly larger changeset.

Just to make sure you're not doing too much work upfront: if the changes are related to TypeScript, please be aware that culori is intentionally a JavaScript, not a TypeScript, project. (One TypeScripty thing we _can_ do in this repository is maintain a `t.ds` typedef file, if that is helpful.)

**Comment by akx** [comment]
> > this is a prelude for an, ahem, slightly larger changeset.
> 
> Just to make sure you're not doing too much work upfront: if the changes are related to TypeScript, please be aware that culori is intentionally a JavaScript, not a TypeScript, project. (One TypeScripty thing we _can_ do in this repository is maintain a `t.ds` typedef file, if that is helpful.)

You may have guessed right... 😅 What's the reason for not being (or allowing being) TypeScript? The built umd/cjs library would be (nearly - I checked) exactly the same, and compiled ESM modules could also be shipped in NPM packages with no sweat.

**Comment by danburzo** [comment]
> What's the reason for not being (or allowing being) TypeScript?

In a nutshell: the various trade-offs of a change in tech/tooling do not currently offer a net benefit to this project. We made the aesthetic choice of keeping everything as simple as possible, with minimal tooling.

**Comment by akx** [comment]
@danburzo Okay, I can dig that, no worries. If you want to take a peek, there's a mostly-complete conversion over at https://github.com/Evercoder/culori/compare/master...akx:ts :)

I'll see if I could neatly extract .d.ts out of that – I originally started this to get better Culori interop for my https://github.com/akx/gradient project (and there's a small partial .d.ts in there).

**Pull request merged**